puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/puppet-deploy.yaml

```diff
@@ -6,6 +6,8 @@
           url: https://github.com/theforeman/foreman-infra
     triggers:
       - github
+      - pollscm
```

Comment:
I think this needs a `:` to be valid yaml

Reply:
oops good catch!